src/server/script_mgr.cc

```diff
@@ -38,6 +38,12 @@ ABSL_FLAG(bool, lua_allow_undeclared_auto_correct, false,
           "access undeclared keys, automaticaly set the script flag to be able to run with "
           "undeclared key.");
 
+ABSL_FLAG(
+    std::vector<std::string>, lua_undeclared_keys_shas,
+    {"351130589c64523cb98978dc32c64173a31244f3"},  // Sidekiq's scheduler script, see #2442
```

Comment:
please also add `6ae15ef4678593dc61f991c9953722d67d822776`

Reply:
Please save me the trouble of looking up what this is, and tell me what I should write in the comment? :pray: :rofl:

Reply:
sidekiq as well, different script